WooCommerce/Classes/Authentication/Application Password/ApplicationPasswordAuthorizationViewModel.swift

```diff
@@ -4,7 +4,7 @@ import Yosemite
 /// View model for `ApplicationPasswordAuthorizationWebViewController`.
 ///
 final class ApplicationPasswordAuthorizationViewModel {
-    private let siteURL: String
+    let siteURL: String
```

Comment:
Seems like this change is not necessary?

Reply:
The function `applicationPasswordDisabledUI` under `navigateToErrorUI` uses the `siteURL`, so it needs to be declared as internal now.